abstractions/php/src/HttpCore.php

```diff
@@ -0,0 +1,9 @@
+<?php
```

Comment:
Some initial thoughts.

Should we use this client implementation as our HTTP Core abstraction here instead of in Graph Core?

https://github.com/microsoftgraph/msgraph-sdk-php-core/blob/philip/feat/http-client-factory/src/Http/HttpClientInterface.php